originals/s.yaml

```diff
@@ -505,6 +505,26 @@
     genres:
     - Simulation
 
+- name: SimCity 4
+  names:
+  - 'SimCity 4: Rush Hour'
+  - SimCity 4 Deluxe Edition
+  external:
+    website: 'https://www.ea.com/games/simcity/simcity-4'
+    wikipedia: SimCity 4
+  platforms:
+  - OSX
+  - Windows
+  meta:
+    genres:
+    - Real-Time Strategy
```

Comment:
I think this genre should be removed. True it is a real time and also a strategy game, but the "RTS" term is used for a more specific kind of game, typically involving base building, unit control, and resource gathering.

Reply:
That makes sense. I'll change that now. Genres can be a bit confusing regarding what 'technically' fits in one and what's generally understood to be part of the genre.